### DISCUSSION TITLE
Welcome to Vython Discussions!

### DISCUSSION THREAD

**Discussion by dopevog** [cvython/Vython#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask __questions__ you’re wondering about.
  * Share __ideas__.
  * Engage with other __community members__.
  * Welcome others and are open-minded. Remember that this is a community we
  build __together__ 💪.

**Comment by fineans** [comment]
Hello!
